### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by alcohol** [composer/satis#403]
I want to move some of the docs from https://getcomposer.org/doc/articles/handling-private-packages-with-satis.md into the `docs` directory, and improve them. Currently they are not easy to find, not very precise/detailed and overal just not great (in my opinion).

Any help, thoughts and feedback is appreciated.

**Comment by timurgafarov** [comment]
That's a good idea. I can write a short tutorial if you want.

**Comment by alcohol** [comment]
Yeah I made a start, you can see results so far at https://composer.github.io/satis/ (this is based on contents of `/docs/` folder). Feel free to submit a PR against said directory.
